### DISCUSSION TITLE
Week 8 - S2

### DISCUSSION THREAD

**Discussion by auroranavas** [ISPP-G5/NexONG#186] [Knowledge Database]
### FEEDBACK SOBRE LA PRESENTACIÓN
- **Evaluación:** Indicar en el rendimiento global la fórmula usada. Además, no se ve que la fórmula actual sea efectiva.
- **Problemas:** Poner el seguimiento de las soluciones. Considerar unificar problemas con retrospectiva.
- **ALM:** Leyenda de Codacy no se ve bien.
- **Revisiones:** Dedicar más tiempo a ellas
- Tener la **demo en local** y **ordenador de repuesto**.

**Comment by fracalrod3** [comment]
> ### FEEDBACK SOBRE LA PRESENTACIÓN
> * **Evaluación:** Indicar en el rendimiento global la fórmula usada. Además, no se ve que la fórmula actual sea efectiva.
> * **Problemas:** Poner el seguimiento de las soluciones. Considerar unificar problemas con retrospectiva.
> * **ALM:** Leyenda de Codacy no se ve bien.
> * **Revisiones:** Dedicar más tiempo a ellas
> * Tener la **demo en local** y **ordenador de repuesto**.

No ha dicho que le dediquemos más tiempo, sino que quede claro que estamos haciendo un esfuerzo para mejorar el número de revisiones y solucionando los problemas que había de poca cantidad.

Lo digo porque después de la presentación le comenté que simplemente no se tarda lo mismo en hacer una revisión que en realizar el código o redactar el documento, siendo que llevamos 83h (aprox.) de revisión, y me respondió Müller que sí, que es lo normal y que simplemente era hacer hincapié en que se está revisando más ya que dijimos que había problemas en el primer Sprint.

**Comment by marnunrey2** [comment]
Algun feedback mas:
- falta diapositiva de las horas de cada persona
- mencionar como solucionar los problemas con los usuarios pilotos
- muy bien el anuncio y haberlo unido con el killer opener
- muy buena reaccion de lo q apareceria en la demo
